Makefile

```diff
@@ -151,6 +154,21 @@ test-integration: $(BINS)
 	cp akashctl akashd ./_build
 	go test -mod=readonly -p 4 -tags "integration $(BUILD_MAINNET)" -v ./integration/...
 
+test-e2e-integration: export KIND_K8S_IP=$(KIND_API_IP)
+test-e2e-integration: export KIND_APP_PORT=$(KIND_HTTP_PORT)
```

Comment:
There are two things about this that I didn't know were possible.  I still don't get how the `export X=Y` dependency thing works.

Choose a method and use it consistently.  Either this magic or the more standard way of doing things:

```
KIND_VARS = FOO=x BAR=y
# ...

test-foo:
    $(KIND_VARS) go test foo
test-bar
    $(KIND_VARS) go test bar
```

Reply:
Yeah, I agree this is weird wizardry, the reason I switched it back to this is that the envvar is scoped to only the `test-e2e-integration` target. Cuts down on variables being defined globally, but only used in one place. 

Second reason is that when I prefixed the envvars inline to the command; `ENVV=foo go test...` the GH action's stage output never seemed to return testing output until the full duration of test, which added to the PITA feedback loop... So it doesn't matter when things are working, but increases debugging agony a bit.

I like your KIND_VARS prefix grouping better though, it makes adding new targets possible with less boilerplate.